packages/atomic/src/components/common/search-box/search-box-common.tsx

```diff
@@ -91,7 +91,13 @@ export class SearchBoxCommon {
     item.query && this.props.clearSuggestions();
   }
 
-  public getSearchInputLabel() {
+  public getSearchInputLabel(minimumQueryLength = 0) {
```

Comment:
Not very happy with having to pass in `minimumQueryLength` param which is a child class only prop into the parent class func. Let me know of any suggestions..

Reply:
Nothing quick come to mind... 🤔